Scripts/script-DemistoSendInvite.yml

```diff
@@ -0,0 +1,57 @@
+commonfields:
+  id: DemistoSendInvite
+  version: -1
+name: DemistoSendInvite
+script: |-
+  var res = executeCommand('demisto-api-get', {url: 'roles'});
+  if (res[0].Type == 4) {
+      throw res[0].Contents;
+  }
+
+  var roles = res[0].Contents.response;
+
+  var roleIDs = [];
+  var roleNames = args.roles.split(',');
+
+
+  roleNames.forEach(function(roleName) {
+      var roleID = dq({roles:roles},'roles(val.name=="'+roleName+'").id');
+      if (!roleID) {
+          throw 'Cannot find role ' + roleName;
+      }
+      roleIDs.push(roleID[0]);
+  });
+
+  body = {
+      email: args.email,
+      roles: roleIDs
+  };
+
+  res = executeCommand('demisto-api-post', {url: 'invite', body: body});
+  if (res[0].Type == 4) {
```

Comment:
We also have the isError function you can use.. looks better and more readable, if you change please change in all places.